linux_files/setup

```diff
@@ -875,14 +880,16 @@ if (whiptail --title "DOCKER" --yesno "Would you like to install the bridge to D
 
     docker-compose version
 
-    if [[ $(wslpath 'C:\\') != '/c/' ]]; then
+    if [[ $(wslpath 'C:\\') = '/mnt/c/' ]]; then
```

Comment:
why would you need to do this check?
just do `wslpath /` and that should be your wsl defined root (based on wsl.conf)?